3.processing_features/scripts/2.normalize_sc.py

```diff
@@ -8,42 +8,46 @@
 # In[1]:
 
 
-import sys
 import pathlib
-import os
 import yaml
-import json
+import pprint
 
 import pandas as pd
 from pycytominer import normalize
 from pycytominer.cyto_utils import output
 
 
+# ## Set paths and load in dictionary from annotated run
+
 # In[2]:
 
 
 # output directory for normalized data
 output_dir = pathlib.Path("./data/normalized_data")
-# if directory if doesn't exist, will not raise error if it already exists
```

Comment:
slight typo

Reply:
That is a typo in a code comment I deleted, so no worries there.